### DISCUSSION TITLE
What is better: PHP or getting eaten by a lion?

### DISCUSSION THREAD

**Discussion by samuelmarina** [samuelmarina/is-even#68] [General]
## 👾 PHP vs Getting eaten by a lion! 🦁
I just found out that we can create discussions here in GitHub, so I decided to start one. Please:
  * Write your answer to the question
  * Give strong arguments
  * Engage with other people, you don't have to be nice tho
  * Do not use plastic straws when you go to Starbucks

  Alright, let the fight begin

**Comment by Aworldc** [comment]
Getting eaten by a lion... lol.
long live node.js

**Comment by SarangT123** [comment]
Getting eaten by a lion 
At least I'm saving a species instead of killing one

**Reply by cremble-cloud** [reply]
OHHHHHHHHHH

**Comment by Niknu** [comment]
Why has this project been created?
It seems pretty weird from a c++ perspective xD

**Reply by adi-g15** [reply]
To help enthusistic kids so they don't need to find whether a number is even or not, instead of playing Mario for some time... it saves their fun time :smile:

**Comment by Shashank3736** [comment]
Learning PHP

My life is full of wrong decisions one more will not make a diff.

**Comment by karansingh-in** [comment]
Getting eaten by a lion, 

coz I've no idea about PHP except for the fact that PHP sucks

**Comment by robsonvpires** [comment]
Lion thing seems a little less painful

**Comment by shoshanabezalel** [comment]
PHP getting eaten by a lion
